Plugins/JiraCommitHintPlugin/JiraCommitHintPlugin.cs

```diff
@@ -282,6 +290,17 @@ private async Task<JiraTaskDTO[]> GetMessageToCommitAsync(Jira jira, string quer
             }
         }
 
+        private Jira createJiraClient(string url, string username, string password, string authType)
+        {
+            if (JWT.Equals(authType))
```

Comment:
This is more idiomatic:
```suggestion
            if (JWT == authType)
```